File: 2024/atividade_C/atividade05.c
#include <stdio.h>

int num,tipo;
int main()
{
    printf("Informe um número inteiro: \n");
    scanf("%d", &num);
    for(int i=1; i<=num; i++)
    {
        tipo = i % 2;
        if(tipo == 0)
        {
            printf("%d Par\n",i);
        }
        else
        {
            printf("%d Ímpar\n",i);
        }
    }
return 0;
}

File: 2024/atividade_py/atividade05.py
print("Informe um número inteiro: ", end="");
num = int(input());

for i in range(1,num+1):
    
    tipo = i % 2;
    
    if(tipo == 0):
        print(f"{i} Par");
    else:
        print(f"{i} Ímpar");

